### ISSUE TITLE
Can't trigger chosen_inline_result flavor after sending audio via inline

### ISSUE THREAD

**Issue by madhattermonroe** [nickoala/telepot#282]
`def on_inline_query(self, msg):
    def compute_answer():
            query_id, from_id, query_string = telepot.glance(msg, flavor='inline_query')
            audio = [InlineQueryResultAudio(
                type="audio",
                id="1",
                title='Ermil',
                performer="Yandex SpeechKit",
                audio_url="{}/generate?key={}&text={}&speaker={}&quality=hi&format=wav" \
                    .format(HOST, YandexApiKey, quote(query_string, safe=""), YandexApiSpeakerMan)
            ),
                InlineQueryResultAudio(
                    type="audio",
                    id="2",
                    title='Jane',
                    performer="Yandex SpeechKit",
                    audio_url="{}/generate?key={}&text={}&speaker={}&quality=hi&format=wav" \
                        .format(HOST, YandexApiKey, quote(query_string, safe=""), YandexApiSpeakerWoman)
                )
            ]
            return audio
        self.answerer.answer(msg, compute_answer)
`

and just test method:
`    def on_chosen_inline_result(self, msg):
        from pprint import pprint
        pprint(msg)
        result_id, from_id, query_string = telepot.glance(msg, flavor='chosen_inline_result')
        print(self.id, ':', 'Chosen Inline Result:', result_id, from_id, query_string)
`

After choosing voice message doesn't send to other people.

**Comment by garciaw** [comment]
Did you enable 100% inline feedback with @botfather? I don't remember what the default is, but I believe it's not 100%. This means you will only receive a `chosen_inline_result` update for a small percentage of the total.

**Comment by madhattermonroe** [comment]
Yes, i did.  I guess... maybe if I want to send audio, I need to cache it on Telegram servers?

**Comment by nickoala** [comment]
The [docs](https://core.telegram.org/bots/api#inlinequeryresultaudio) mentions that:

> This will only work in Telegram versions released after 9 April, 2016. Older clients will ignore them.

Could this be the reason?
